storage/memoized_iterator_test.go

```diff
@@ -16,25 +16,36 @@ package storage
 import (
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
+	"github.com/prometheus/prometheus/model/histogram"
 	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/tsdb/tsdbutil"
+
+	"github.com/stretchr/testify/require"
```

Comment:
We usually do import packages outside of github.com/prometheus/prometheus in a block _before_ the block with imports from github.com/prometheus/prometheus (as it was before).